test/relations.jl

```diff
@@ -516,6 +517,36 @@ end
         @test all(saturated.(param_set, ts[RH_sat_mask]))
         @test !any(saturated.(param_set, ts[RH_unsat_mask]))
 
+        # test Clausius Clapeyron relation
+        k = findfirst(q -> q > 0.01, q_tot) # test for one value with q_tot above some threshhold
+        ts_sol = TD.PhaseEquil_ρTq(param_set, ρ[k], T[k], q_tot[k])
+
+        function q_vap_sat(_T::FT) where {FT}
+            _ρ = TD.air_density(param_set, ts_sol)
+            _q_tot = TD.total_specific_humidity(param_set, ts_sol)
+            _phase_type = PhaseEquil{FT}
+            _q_pt = PhasePartition_equil(
+                param_set,
+                _T,
+                oftype(_T, _ρ),
+                oftype(_T, _q_tot),
+                _phase_type,
+            )
+            return TD.q_vap_saturation(
+                param_set,
+                _T,
+                oftype(_T, _ρ),
+                _phase_type,
+                _q_pt,
+            )
+        end
+        ∂q_vap_sat_∂T_fd = _T -> ForwardDiff.derivative(q_vap_sat, _T)
+        @test isapprox(
+            ∂q_vap_sat_∂T_fd(ts_sol.T),
+            TD.∂q_vap_sat_∂T(param_set, ts_sol);
+            atol = 6.1e-3,
```

Comment:
Can we make this an `rtol`?

Reply:
We can, but ouch, this it looks like it passes at `rtol = 0.9` but fails at `rtol = 0.8`. Am I interpreting that right that it's an 80% error? I'm wondering if I did something wrong somewhere

Reply:
Maybe plot both (analytical and FD) to be sure? Identifying those types of issues is what tests are for :)

Reply:
Plotting showed good agreement after all, is it worth adding that plot to the docs? Or is the (log) test sufficient?

Reply:
It's nice to have the plots, I think.